Physics/ContactCallbacks/ContactEvents.cs

```diff
@@ -0,0 +1,462 @@
+﻿using System.Diagnostics;
```

Comment:
Shouldnt this gave have the "this is part of BePuPhysics tutorial" on top?

Reply:
also maybe lets keey all the bepu stuff in one folder

Reply:
I've been thinking about that as well and that was initially the plan, but as we're adding more and more stuff into physics project (some of them taken from bepu and some not) it becomes a mess to keep all of that in one place. I think we'd be better off having separate folders for Collisions / ContactCallbacks / RayCasting.
To make licensing issues as straight as possible I'll move the readme with all the disclaimers up to the root of physics project

Reply:
Let me know if that's a satisfactory solution

Reply:
it is